### PR TITLE
Add file size checker

### PR DESCRIPTION
**Pull request by seqizz**

### CODE REVIEW

File: src/check_logfilesize.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    elif warning_count > 0:
+        print("Warning")
+        print(files)
+        sys.exit(1)
+    else:
+        print("OK")
+        sys.exit(0)
+
+
+def file_size_mb(filePath): return os.path.getsize(filePath) / (1024 * 1024)
```
Comment (hasegeli) [inline src/check_logfilesize.py]: PEP8 says "don't do it if you are not trying to hide anything".

Comment (hasegeli) [reply src/check_logfilesize.py]: You may also write `1024**2`.

File: src/check_logfilesize.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+args = parser.parse_args()
+
+def main():
+    critical_count = 0
+    warning_count = 0
+    files = ""
+
+    for file in os.listdir(args.path):
+        if file.endswith(".log"):
+            current_file_size = file_size_mb(args.path+"/"+file)
```
Comment (hasegeli) [inline src/check_logfilesize.py]: Spaces around opeators

File: src/check_logfilesize.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import argparse
+
+parser = argparse.ArgumentParser(
+    description="A small nagios script to check "
+                "for size of logfiles in given directory")
+
+parser.add_argument("-p", dest="path", help="Define the directory in which the check should run")
+parser.add_argument("-w", dest="warning", help="Define the warning threshold in MB")
+parser.add_argument("-c", dest="critical", help="Define the critical threshold in MB")
+args = parser.parse_args()
```
Comment (hasegeli) [inline src/check_logfilesize.py]: It would be nice to make this `parse_args()` function and call from `main()`.

File: src/check_logfilesize.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+def main():
+    critical_count = 0
+    warning_count = 0
+    files = ""
+
+    for file in os.listdir(args.path):
```
Comment (hasegeli) [inline src/check_logfilesize.py]: `file` might be reserved.  `filename` might be better.

**Comment by seqizz** [comment]
Should be better now

**Pull request merged**